modules/user_accounts/php/edit_user.class.inc

```diff
@@ -454,7 +454,6 @@ class Edit_User extends \NDB_Form
             $user    = \User::factory($this->identifier);
             $success = $user->update($set);
         }
```

Comment:
Made this change because travis was failing because of the extra line.

Reply:
#4688